recipe/meta.yaml

```diff
@@ -46,13 +42,12 @@ requirements:
     - pkg-config    # [osx]
   host:
     - python
-    - setuptools
     - cython
```

Comment:
`Cython>=0.29.18,<3.0`
https://github.com/scipy/scipy/blob/v1.7.3/pyproject.toml#L14

Reply:
Thanks, I'll add the pinning